__tests__/app.tests.js

```diff
@@ -45,6 +45,49 @@ describe("GET /api", () => {
         .then((response) => {
             const body = response.body;
             expect(body).toEqual(endpoints);
-        })
-    })
-})
\ No newline at end of file
+        });
+    });
+});
+
+describe("GET /api/articles/:article_id", () => {
+    test("returns status code 200 & requested article", () => {
+        return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then((response) => {
+            const articles = response.body;
+            expect(articles.article_id).toBe(1);
+            expect(articles).toMatchObject({
+                article_id: expect.any(Number),
+                author: expect.any(String),
+                title: expect.any(String),
+                body: expect.any(String),
+                topic: expect.any(String),
+                created_at: expect.any(String),
+                votes: expect.any(Number),
+                article_img_url: expect.any(String)
+            });
+        });
+    });
+    test("returns status code 404 Not found for non-existent article id", () => {
+        return request(app)
+        .get("/api/articles/9999")
+        .expect(404)
+        .then((response) => {
+            const body = response.body;
```

Comment:
A suggestion not a 'must do' but this is so close to destructuring that I think you'd have a better time using it. As an example we know that everything is sent back on a key of body. You could in your then block use ```{body}``` to represent that. You don't need to test that a property of msg exists AND its contents. You can assert both by asserting that the ```body.msg``` IS a string of something. This proves the key exists and the value.